### PR TITLE
Top.Mail.Ru as AMP-analytics vendor

### PR DESCRIPTION
**Pull request by ilnarb**
PR adds AMP-analytics vendor config for Top.Mail.Ru.
Top.Mail.Ru is a web stat and analytics system.

### CODE REVIEW

**Comment by ilnarb** [comment]
Hi, guys!
When can we expect a review and release?

File: extensions/amp-analytics/0.1/test/vendor-requests.json [hunk 1]
```diff
@@ -222,4 +222,11 @@
     "base": "https://ibeat.indiatimes.com/iBeat/pageTrendlogAmp.html",
     "pageview": "https://ibeat.indiatimes.com/iBeat/pageTrendlogAmp.html?&h=$h&d=$h&url=$url&k=$key&ts=$time&ch=$channel&sid=$uid&at=$agentType&ref=_document_referrer_&aid=$aid&loc=1&ct=1&cat=$cat&scat=$scat&ac=1&tg=$tags&ctids=$catIds&pts=$pagePublishTime&auth=$author&pos=$position&iBeatField=$ibeatFields&cid=_client_id_",
   },
+  "topmailru": {
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/test/vendor-requests.json]: Please place "topmailru" after "snowplow". I know we fail to sort them by alphabetic order somehow(we need to fix it), but the idea is to place them by alphabetic order.

File: extensions/amp-analytics/0.1/vendors.js [hunk 2]
```diff
@@ -1423,6 +1423,39 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
       },
     },
   },
+
+  'topmailru': {
+    'transport': {
+      'beacon': false,
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors.js]: Just to confirm with you that your server does not support `beacon` `xhrpost`? They are recommended.

Comment (ilnarb) [inline extensions/amp-analytics/0.1/vendors.js]: We handle only GET requests now.

File: extensions/amp-analytics/0.1/test/vendor-requests.json (hunk 1 above)
Comment (ilnarb) [inline extensions/amp-analytics/0.1/test/vendor-requests.json]: done

**Comment by ilnarb** [comment]
@zhouyx Thanks for the review. I made required changes.

**Comment by zhouyx** [comment]
Thanks for the Pull Request!
LGTM. Merged.

**Pull request merged**